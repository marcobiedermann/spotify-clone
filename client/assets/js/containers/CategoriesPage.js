import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { fetchCategories } from '../actions/browse';
import CategoriesPage from '../pages/Browse/Categories';

const mapStateToProps = state => ({
  ...state,
  categories: state.browse.categories,
});

const mapDispatchToProps = dispatch => bindActionCreators(
  {
    fetchCategories,
  },
  dispatch,
);

export default connect(
  mapStateToProps,
  mapDispatchToProps,
)(CategoriesPage);
