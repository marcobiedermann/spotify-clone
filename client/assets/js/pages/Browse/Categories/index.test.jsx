import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import CategoriesPage from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter><CategoriesPage /></MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
