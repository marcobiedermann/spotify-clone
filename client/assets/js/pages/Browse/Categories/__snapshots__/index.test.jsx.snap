// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <ul
    className="categories"
  />
</div>
`;
